client/data/settings/resolvers.js

```diff
@@ -22,9 +22,39 @@ export function* getSettings() {
 	try {
 		const result = yield apiFetch( { path } );
 		yield updateSettings( result );
+
+		const surveyResult = yield apiFetch( {
+			path: `${ NAMESPACE }/survey`,
+			method: 'POST',
+			data: {
+				site_id: 999999999,
```

Comment:
I just hardcoded everything in the frontend - obvs some of this will need to be done by the backend, probably the frontend should only submit the `survey_responses` data.